src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
```

Comment:
The JSDoc comment is incomplete. It should describe the function's purpose, parameters, and return value more clearly. For example, you could specify that the function returns a closure that accumulates values and returns the sum when called without arguments.